app/services/base-service.php

```diff
@@ -0,0 +1,5 @@
+<?php
+namespace Elementor\App\Services;
+
+abstract class Base_Service implements Service_Interface {
```

Comment:
Agreed, one of them is also redundant in my opinion.
Not sure which is the best, again you need to test it with the pro implementation. What will happen in the Pro if you want to implement the interface or extend the base which will not be available since to user only updated the pro?